views/show.erubis

```diff
@@ -10,7 +10,7 @@
 
       <ul class="list list-bullet">
         <% @case.files.each do |file| %>
-          <li><a href="<%= file.s3_url %>" title="<%= file.name %>"><%= file.name %></a></li>
+          <li><a href="<%== file.s3_url %>" title="<%== file.name %>"><%== file.name %></a></li>
```

Comment:
I thought <%== meant "display this string raw (i.e. don't html escape it). Isn't that the opposite of what we want?